### PR TITLE
[issue-647] Address extra dependencies in catalog document

### LINKED ISSUE #647: Address dependencies in catalog document
**Problem description**
Need to address the dependencies that are required to use the Table API and catalog in Pravega catalog document to increase readability.

**Problem location**
catalog doc

**Suggestions for an improvement**

### PR DESCRIPTION
**Pull request by fyang86**
Signed-off-by: Fan, Yang <fan.yang5@emc.com>

**Change log description**
- Address the Schema Registry dependency required to use the Table API and catalog in Pravega catalog document.

**Purpose of the change**
Fix #647 

**What the code does**
- Address the Schema Registry dependency in Catalog doc

**How to verify it**
No special verification is required, build should pass though

### CODE REVIEW

File: documentation/src/docs/catalog.md [hunk 1]
```diff
@@ -30,7 +63,7 @@ Pravega uses terms such as *streams* and *scopes* for managing streaming data, b
 
 When using Schema Registry serialization, further information is required in order to describe how to map binary data onto table columns. We have introduced an internal format factory `PravegaRegistryFormatFactory`. Currently it supports Json and Avro formats without any additional encryption and compression codecs.
 
-With the help of schema registry service, it is feasible to map Pravega streams to Flink tables as the following table shows:
+With the help of schema Registry service, it is feasible to map Pravega streams to Flink tables as the following table shows:
```
Comment (crazyzhou) [inline documentation/src/docs/catalog.md:66]: Schema here should also be capitalized.

File: documentation/src/docs/catalog.md [hunk 2]
```diff
@@ -30,7 +63,7 @@ Pravega uses terms such as *streams* and *scopes* for managing streaming data, b
...
 
 | Flink Catalog terms                  | Pravega terms     |
 |--------------------------------------|-------------------|
```
Comment (crazyzhou) [inline documentation/src/docs/catalog.md:69]: I think in the below table, the word `name` is redundant.

**Review by crazyzhou — CHANGES_REQUESTED** [review]

File: documentation/src/docs/catalog.md (hunk 1 above)
Comment (fyang86) [reply documentation/src/docs/catalog.md]: fixed

File: documentation/src/docs/catalog.md (hunk 2 above)
Comment (fyang86) [reply documentation/src/docs/catalog.md]: fixed

File: documentation/src/docs/catalog.md [hunk 3]
```diff
@@ -30,13 +63,13 @@ Pravega uses terms such as *streams* and *scopes* for managing streaming data, b
...
+With the help of Schema Registry service, it is feasible to map Pravega streams to Flink tables as the following table shows:
 
-| Flink Catalog terms                  | Pravega terms     |
-|--------------------------------------|-------------------|
-| catalog name (defined in Flink only) | N/A               |
-| database name                        | scope name        |
-| table name                           | stream name       |
+| Flink Catalog terms                  | Pravega terms |
+|--------------------------------------|---------------|
+| catalog name (defined in Flink only) | N/A           |
```
Comment (crazyzhou) [inline documentation/src/docs/catalog.md:70]: Should be Catalog, but we can also delete this line.

**Review by crazyzhou — APPROVED** [review]

**Pull request merged**